dev/AppLifecycle/AppInstance.cpp

```diff
@@ -398,7 +398,8 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
 
     bool AppInstance::TrySetKey(std::wstring const& key)
     {
-        std::wstring mutexName = m_moduleName + L"_Mutex";
+        auto escapedKey = std::regex_replace(key, std::wregex(L"\\\\"), L"_");
+        std::wstring mutexName = wil::str_printf<std::wstring>(L"%s_%s_Mutex", m_moduleName.c_str(), escapedKey.c_str());
```

Comment:
Is there a limit on maxlength of a named mutex?

Not worried about it short-term but if there is a limit it _may_ be something we want to consider dealing with longer term